projects/ngrx.io/package.json

```diff
@@ -11,7 +11,7 @@
     "aio-use-npm": "node tools/ng-packages-installer restore .",
     "aio-check-local": "node tools/ng-packages-installer check .",
     "ng": "yarn check-env && ng",
-    "start": "yarn check-env && ng serve --configuration=fast",
+    "start": "yarn check-env && ng serve --configuration=stable --host 0.0.0.0",
```

Comment:
Instead of changing this script, create a `start:docker` script and use that in the container

Reply:
added `start:docker` on my new commit